hack/update-generated-crd-code.sh

```diff
@@ -52,4 +57,25 @@ controller-gen \
   paths=./pkg/controller/... \
   output:crd:artifacts:config=config/crd/bases
 
+# Add lables and annotations for CRDs by kustomize
+# The controller-tool is not prefer to do this https://github.com/kubernetes-sigs/controller-tools/issues/454
+pushd config/crd/bases
+for crd in *.yaml; do
+  echo $crd
+  cat <<EOF > kustomization.yaml
+  apiVersion: kustomize.config.k8s.io/v1beta1
+  kind: Kustomization
+  resources:
+  - $crd
+  commonLabels:
+    app.kubernetes.io/name: velero
+  commonAnnotations:
```

Comment:
What if the user isn't installing via Helm? Then we're annotating the CRDs with Helm information, but it's not actually used.